### ISSUE TITLE
Compatibility View in IE8

### ISSUE THREAD

**Issue by davestewart** [metafizzy/packery#188]
Not so much a bug, more something that might be worth adding to the docs or even erroring more gracefully on, but Packery was working perfectly on IE 8 for weeks, then on the final debug and deploy, it started to break.

It took me about an hour to track down the bug, working in a VM, then I eventually got it. 

IE had somehow gone into compatibility mode, so was behaving like IE7 (even with a [metatag](http://stackoverflow.com/questions/10445781/using-meta-http-equiv-x-ua-compatible-content-ie-8-mode-in-sites-good-pr) to ask it not to) so Element.prototype was not available.

It might be worth adding some kind of check for this so that the minified version doesn't choke without giving a bit more information. IE8 is our lowest supported browser, and not fun to debug - It was a bit of a detective hunt and piecing together the clues to work this out!

Also, we're trying to pay for the plugin right now, but the credit card page is issuing a Service Error. We'll try again later, but you might want to check it out.

Cheers,
Dave Stewart

**Comment by desandro** [comment]
Compatibility view in IE8 is a mysterious thing. I haven't been been able to identify just what sets it off, if its even Packery.

**Comment by davestewart** [comment]
Surely a check for window.Element && window.Element.prototype would work for starters though?

**Comment by desandro** [comment]
That's a just the symptom. If IE8 is in Compatibility mode other things will break as well.

**Comment by davestewart** [comment]
Sorry - to clarify: the project we deployed, which suddenly broke in IE had 1000s of lines of minified code, and it was a lot of effort to track the bug down, in the end, to Packery. Of all the 1000s of lines in the entire project of custom JS, other libs, etc, etc, Packery was the *only* library to fail on IE - however getting to that conclusion was a herculean effort of removing code, running IE in a VM, working with IE8s not so great developer tools, etc. etc.

It seems to me that if Packery is going to fail anyway, having some kind of IE8 compatibility mode check on instantiation, and outputting a *meaningful* error message such as "Packery does not run in IE compatibility mode", or even putting this in the FAQ, would save developers a lot of time on something which it turns out is IE8/Packery-specific.

Not really sure how I can put the point across any clearer, so I made myself understood.

Thanks,
Dave
